backend/src/main/resources/db-migrations/h2/v0.1-[vendor]-data-init.sql

```diff
@@ -30,4 +30,5 @@ VALUES
 INSERT INTO `confirmation_code` (`id`, `code`, `expiry_date`, `user_id`, `created_by`, `created_date`, `last_modified_by`, `last_modified_date`)
 VALUES
 	('a6ea4925-f813-493e-80ec-a5b90ca28b6c', '12345', CURRENT_TIMESTAMP, 'f9f33652-0ebd-46bc-8d93-04cef538a689', 'flyway-community-edition', CURRENT_TIMESTAMP, 'flyway-community-edition', CURRENT_TIMESTAMP),
+	('a6ea4925-f813-493e-80ec-a5b90ca28aaa', '54321', CURRENT_TIMESTAMP-100, 'f9f33652-0ebd-46bc-8d93-04cef538a689', 'flyway-community-edition', CURRENT_TIMESTAMP, 'flyway-community-edition', CURRENT_TIMESTAMP),
```

Comment:
**Massive nitpick:** Since spaces are used around operands in the line below, for consistency:

```suggestion
	('a6ea4925-f813-493e-80ec-a5b90ca28aaa', '54321', CURRENT_TIMESTAMP - 100, 'f9f33652-0ebd-46bc-8d93-04cef538a689', 'flyway-community-edition', CURRENT_TIMESTAMP, 'flyway-community-edition', CURRENT_TIMESTAMP),
```